src/Repository/User/UserRepositoryInterface.php

```diff
@@ -17,4 +17,9 @@ public function createWith(string $email, string $username): User;
     public function save(User $user): void;
 
     public function remove(User $user): void;
+
+    public function updateUserInformation(User $user, ?string $emailAddress = null, ?string $username = null): void;
```

Comment:
is the email really nullable?

Reply:
for the `?string`?  at first read I believed it was about the `= null`, sorry will fix that.
On second thought it might be better to have a function to update username separated from the one updating the email address, @tdutrion your thoughts on this idea?

Reply:
That's up to you. The username and email being mandatory all along, I'd probably not allow nullable but just feed in the original data.